lib/config/openssh/openssh.go

```diff
@@ -68,7 +75,9 @@ type SSHConfigParameters struct {
 	ProxyHost           string
 	ProxyPort           string
 	ExecutablePath      string
+	Username            string
 	DestinationDir      string
+	Port                int
```

Comment:
I feel like we might want to explain that `Port` here is the default port that will be used if no other port is specified. I can see this getting confusing with `ProxyPort` in the struxt as well.